.github/labeler.yml

```
@@ -1,10 +1,35 @@
-# Add 'zkevm-circuits' label to any change to files within the zkevm-circuits dir EXCEPT for the evm_circuit sub-folder
-zkevm-circuits:
-  - any: ['zkevm-circuits/src/**/*', '!zkevm-circuits/src/evm_circuit/**/*']
-# Add 'opcodes' to any changes within 'evm_circuit' folder or any subfolders
-opcode:
-  - zkevm-circuits/src/evm_circuit/**/*
-bus-mapping:
+# Add CI label for any CI changes
+CI:
+  - .github/**/*
```

Maybe we should quote this, so yaml doesn't interpret the "." in a strange way? 
```suggestion
  - ".github/**/*"
```

---

Was a good try. But not working neither.

I'm really not following what's going on.. Will take a deeper look